### ISSUE TITLE
Unhandled exception: null

### ISSUE THREAD

**Issue by opk12** [ubipo/osmfocus#22]
##  Exception details
Message: `null`
         
##  What happened / what actions did you take before the bug occurred?
Launch. Point to a housenumber, road, amenity. The app crashes.

##  Comments
Why the shortened class and method names in the stack trace?

## Stack trace
```
i6.a
	at i6.l.h(model.kt:6)
	at i6.h.e(model.kt:4)
	at i6.h.f(model.kt:1)
	at net.pfiers.osmfocus.service.MapApiDownloadManager.getGeometry(MapApiDownloadManager.kt:2)
	at u6.h.q(MapVM.kt:13)
	at u6.h.h(MapVM.kt:1)
	at u6.h$q.i(MapVM.kt:4)
	at w2.a.o(ContinuationImpl.kt:3)
	at u5.a1.run(DispatchedTask.kt:22)
	at kotlinx.coroutines.scheduling.a.y(CoroutineScheduler.kt:1)
	at kotlinx.coroutines.scheduling.a$c.c(CoroutineScheduler.kt:4)
	at kotlinx.coroutines.scheduling.a$c.m(CoroutineScheduler.kt:4)
	at kotlinx.coroutines.scheduling.a$c.run(CoroutineScheduler.kt:1)

```

## System / app info
```
App version: 1.1.2-fdroid
App version code: 112
App build type: release
Android version: 25
Device name: samsung
```

**Comment by ubipo** [comment]
I'm not 100% certain but I'm pretty sure this was caused by computing the geometry of a relation with incomplete members. This is a regression from re-implementing the OSM API parsing code. If my guess is correct then this was fixed in 00dd435 and will be released with v1.2.0.

I'm closing this for now. Do let me know if you can reproduce it after v1.2.0.

**Comment by opk12** [comment]
I cannot reproduce it on `1.2.2-fdroid`. Thanks.
